cmd/clusterctl/client/cluster/mover_test.go

```diff
@@ -2408,3 +2408,48 @@ func TestWaitReadyForMove(t *testing.T) {
 		})
 	}
 }
+
+func Test_applyMutators(t *testing.T) {
```

Comment:
How about using t. Parallel?

Reply:
Not a bad suggestion, but I'd rather not, because
(a) There's no measurable decrease in time for this specific test, and
(b) relatively few tests across the repo use it.